src/SerialGC.h

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) David Ellis
+// This file is part of VLCB-Arduino project on https://github.com/SvenRosvall/VLCB-Arduino
+// Licensed under the Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License.
+// The full licence can be found at: http://creativecommons.org/licenses/by-nc-sa/4.0/
+
+#pragma once
+
+// header files
+#include <Arduino.h>
+#include <Controller.h>
```

Comment:
Do you need to include <Controller.h> ?

Reply:
I copied the includes from CAN2515.h, so didn't check that
However, if I remove it,. the build fails
I suspect CanTransport requires it

Reply:
Yes, CanTransport.h needs it. It should include <Controller.h>. Then it can be removed from here.
Same effect but neater.